configs/ARK_FPV/config.h

```diff
@@ -0,0 +1,201 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU       STM32H743
+#define SYSTEM_HSE_MHZ      16
+#define BOARD_NAME          ARK_FPV
+#define MANUFACTURER_ID     ARKE
+
+#define TARGET_IO_PORTH     0xffff
+#define TARGET_IO_PORTI     0xffff
```

Comment:
Should this be added to [src/platform/](https://github.com/betaflight/betaflight/blob/bbaf961edb3acbf6b4eaf7e3cedb6733fdb1aaa9/src/platform/STM32/target/STM32H743/target.h#L61) ?

Reply:
yeah it should probably be in STM32/target/STM32H743/target.h

Reply:
https://github.com/betaflight/betaflight/pull/14199